src/dotnet-install.ps1

```diff
@@ -277,13 +278,17 @@ function Get-NormalizedChannel([string]$Channel) {
         return ""
     }
 
+    if ($Channel.Contains("Current")) {
+        Say-Error 'Value "Current" was depricated for -Channel option. Please use "STS" instead.'
```

Comment:
Outputting to error stream will fail a lot of build pipelines. How about just a warning?
Typo: deprecated.